extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/handler/WeatherUndergroundHandler.java

```diff
@@ -203,8 +204,8 @@ private void updateChannel(String channelId) {
             State state = null;
             if (value == null) {
                 state = UnDefType.UNDEF;
-            } else if (value instanceof Calendar) {
-                state = new DateTimeType((Calendar) value);
+            } else if (value instanceof ZonedDateTime) {
```

Comment:
are you sure that WeatherUndergroundJsonUtils suddenly returns ZonedDateTimes instead of Calendars? I can't see that you adapted any of the Calendar occurrences in `org.eclipse.smarthome.binding.weatherunderground.internal.json.*`.